azure/scripts/ansible/playbooks/roles/autoscaler/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Get Machine Set
+  ansible.builtin.command: |
+    oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].metadata.labels.machine\.openshift\.io/cluster-api-cluster}'
+    --kubeconfig /home/"{{ BOOTSTRAP_ADMIN_USERNAME }}"/.kube/config
+  register: CLUSTER_ID_OUTPUT
+  changed_when: CLUSTER_ID_OUTPUT.rc != 0
+- name: Set Cluster ID
+  ansible.builtin.set_fact:
+    CLUSTER_ID: "{{ CLUSTER_ID_OUTPUT.stdout | trim }}"
+- name: Execute Cluster Autoscalar
+  kubernetes.core.k8s:
+    state: present
+    kubeconfig: "/home/{{ BOOTSTRAP_ADMIN_USERNAME }}/.kube/config"
+    src: files/cluster-autoscaler.yaml
+    wait: true
+    wait_sleep: 50
+- name: Jinja Conversion for Machine Autoscaler Yaml
+  ansible.builtin.template:
+    src: files/machine-autoscaler.yaml
+    dest: "{{ GIT_CLONE_DIR }}/azure/scripts/ansible/playbooks/roles/ocpcluster/files/machine-autoscaler.yaml"
+    mode: '0644'
+- name: Execute Machine Autoscalar
+  kubernetes.core.k8s:
+    state: present
+    kubeconfig: "/home/{{ BOOTSTRAP_ADMIN_USERNAME }}/.kube/config"
```

Comment:
Shouldn't be necessary.

Reply:
Removed the `kubeconfig`. Working as expected.